apache/docker-entrypoint.sh

```diff
@@ -94,6 +94,21 @@ if  [[ "$1" == apache2* || "$1" == php-fpm || "$1" == bin* ]]; then
       ${ROUNDCUBEMAIL_PLUGINS_SH};
   fi
 
+  if [ ! -d skins/${ROUNDCUBEMAIL_SKIN} ]; then
+    # Installing missing skin
+    echo "Installing missing skin: ${ROUNDCUBEMAIL_SKIN}"
+    composer \
+      --working-dir=/usr/src/roundcubemail/ \
```

Comment:
I just noticed that for the plugins the code is fixed since 5c73c3e4.

You seem to be working on an outdated base of the "master" branch. Could you rebase your changes? Or, if you don't know how to do that, have a look at the commit I linked above and copy the syntax for the `working-dir` line?